Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.6.6'
+ruby '2.6.9'
```

Comment:
This is necessary since Docker's 2.6.6 images run an older Debian versino, which some gems were having trouble with.